connectors/golden-tables/src/test/scala/io/delta/golden/GoldenTables.scala

```diff
@@ -865,6 +866,154 @@ class GoldenTables extends QueryTest with SharedSparkSession {
   }
   */
 
+  for (parquetTimestampType <- SQLConf.ParquetOutputTimestampType.values) {
```

Comment:
Can you add the following configs to avoid generating the CRC and SUCCESS files? They are just bloating the repo.

.set("dfs.client.read.shortcircuit.skip.checksum", "true")
.set("mapreduce.fileoutputcommitter.marksuccessfuljobs", "false")

Reply:
"dfs.client.read.shortcircuit.skip.checksum", "true" didn't do anything seems like it's only for reads. Did a bit of googling didn't find anything that worked